### ISSUE TITLE
Doesn't work in nvm 0.30.2

### ISSUE THREAD

**Issue by uiureo** [koenpunt/capistrano-nvm#12]
https://github.com/koenpunt/capistrano-nvm/blob/master/lib/capistrano/tasks/nvm.cap#L31

In the latest version (**v0.30.2**) of nvm, this `/tmp/.../nvm-exec.sh` doesn't work.

Because `/usr/local/nvm/nvm.sh` removes arguments now:
https://github.com/creationix/nvm/compare/v0.30.1...v0.30.2#diff-e1e8e036966ad5aaf7709712cd5d8ee3R2372

For example: 

nvm-echo.sh:
```sh
#!/bin/bash
source "/usr/local/nvm/nvm.sh"
echo "$@"
```

### nvm 0.30.1
```sh
$ ./nvm-echo.sh node --version
node --version
```

### nvm 0.30.2
```sh
$ ./nvm-echo.sh node --version

# it displays nothing
```

## workaround
You can avoid this issue by setting `nvm_prefix` option:

```ruby
set :nvm_prefix, "#{fetch(:nvm_path)}/nvm-exec"
```

I'll send a pull request later.

**Comment by koenpunt** [comment]
I'm fine with this, but hesitant to merge because without the generated `nvm-exec.sh` it won't work on older installations. 
Besides, the issue looks more like a bug in nvm, because IMO nvm shouldn't touch those arguments..

**Comment by ljharb** [comment]
This should be fixed with v0.31.0.
